### PR TITLE
[WIP] Add wait command

### PR DESCRIPTION
**Pull request by rmweir**

### CODE REVIEW

**Comment by rmweir** [comment]
Use events channel to read status, don't print, exit with same error code if any containers exited with it.

File: cmd/wait/wait.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	if err := validate.NArgs(ctx, "wait", 1, -1); err != nil {
+		return err
+	}
+
+	client, err := cliclient.New(ctx)
+	if err != nil {
+		return err
+	}
+
+	tick := time.Tick(500 * time.Millisecond)
```
Comment (dweomer) [inline cmd/wait/wait.go]: would like to see `tick` closed explicitly

File: cmd/wait/wait.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+	for _, containerID := range ctx.Args().Slice() {
+		select {
+		case <-tick:
```
Comment (dweomer) [inline cmd/wait/wait.go]: should have a case for `<-ctx.Done()` that returns `ctx.Error()`

**Comment by dweomer** [comment]
@rmweir please also include an entry in the "Running" section of the README.md (e.g. `k3c --help` output)

**Pull request closed without merging**